### ISSUE TITLE
Incompatible with chai 3.4.0?

### ISSUE THREAD

**Issue by elado** [chaijs/chai-change#4]
```js
var x = 0;

expect(function() { x += 1; }).to.change(function() { return x });
expect(function() {     }).not.to.change(function() { return x });
```
error:

```
AssertionError: expected [Function] to have a property undefined
at Assertion.assertChanges (node_modules/chai/lib/chai/core/assertions.js:1604:40)
at Assertion.assert (node_modules/chai/lib/chai/utils/addChainableMethod.js:83:49)
```

**Comment by timruffles** [comment]
Thanks for reminding me.

Yes - unfortunately chai added a not great change implementation of its own.

Will put in a helpful error message when used with chai 1.x

**Comment by timruffles** [comment]
Fixed in 2.0.0 :)
